corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -86,6 +86,11 @@ class EntryInstances(PostProcessor):
 
     See docs/apps/instances.rst"""
 
+    IGNORED_INSTANCES = {
+        'jr://instance/remote',
+        'jr://instance/search-input',
+    }
```

Comment:
What does it mean for instances to be ignored in this context? That they're not added to the suite file?

Also, was going to read more but it seems like `docs/apps/instances.rst` has been moved or removed.